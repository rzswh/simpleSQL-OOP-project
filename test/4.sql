CREATE DATABASE OOP;
USE OOP;
CREATE TABLE qaz(wsx DOUBLE NOT NULL, PRIMARY KEY(wsx), edc INT NOT NULL, rfv CHAR);
CREATE TABLE maz(wsx DOUBLE NOT NULL, PRIMARY KEY(wsx), edc INT NOT NULL, rfv CHAR);
CREATE TABLE oaz(wsx DOUBLE NOT NULL, PRIMARY KEY(wsx), edc INT NOT NULL, rfv CHAR);
SHOW columns from qaz;
SHOW TABLES;
DROP TABLE oaz;
CREATE TABLE aaz(wsx DOUBLE, PRIMARY KEY(wsx), edc INT NOT NULL, rfv CHAR);
SHOW TABLES;
DROP DATABASE OOP;
