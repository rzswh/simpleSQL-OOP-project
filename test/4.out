Field	Type	Null	Key	Default	Extra
wsx	double	NO	PRI	NULL	
edc	int(11)	NO		NULL	
rfv	char(1)	YES		NULL	
Tables_in_OOP
maz
oaz
qaz
Tables_in_OOP
aaz
maz
qaz
